content/about.html

```diff
@@ -135,44 +86,66 @@ <h2 class="no-link">
         <div class="container">
           <div class="row">
             <div class="col-sm-12 col-lg-10 col-xl-8">
-              <h2 class="no-link">What's on web.dev?</h2>
+              <h2 class="no-link">Your tools for success</h2>
               <p>
-                If you want to learn how to build on the web, or how to build on
-                the web better, you've come to the right place. A basic
-                knowledge of HTML/CSS/JS is all you need to get started using
-                the tools and guidance on web.dev to:
+                To get started, all you need is a basic understanding of HTML, CSS, and JavaScript.
               </p>
               <div class="whats-on__card-row">
                 <div class="whats-on__card">
                   <h3 class="no-link">Measure</h3>
                   <p>
-                    Using Lighthouse—an open-source, automated tool for running
-                    diagnostics on any web page—will help you understand how and
-                    where to start improving the quality of your web page. It has
-                    audits for performance, accessibility, progressive web apps, and
-                    more.
+                    Audit your site with <a href="https://developers.google.com/web/tools/lighthouse/" target="_blank">Lighthouse</a> 
+                    to get guidance you can use to improve:
                   </p>
+                  <ul>
+                    <li>Performance</li>
+                    <li>Progressive Web App status</li>
+                    <li>Accessibility</li>
+                    <li>Use of best practices</li>
+                    <li>Search engine optimization</li>
+                  </ul>
                   <a href="/measure"
                       class="button button-flat gc-analytics-event"
                       data-category="web.dev"
                       data-label="about, explore measure"
-                      data-action="click">Learn More</a>
+                      data-action="click">Test my site</a>
                 </div>
                 <div class="whats-on__card">
                   <h3 class="no-link">Learn</h3>
                   <p>
-                    Organized around the core principles of what makes a better web,
-                    you'll find a comprehensive collection of actionable guides,
-                    tutorials, and codelabs to help you understand how to make your
-                    site better, and even practice doing it.
+                    Explore topics that matter for the modern web, 
```

Comment:
Suggestion to balance the right side to be aligned better with the left side.

Expand your knowledge and practice new skills for the modern web:

* Load speed
* Network resilience
* Data security
* Site discoverability
* Accessibility for all users

Reply:
This will have both sides aligned, and the main thing that jumps out are the topics and audit categories.